src/tools/rbd_nbd/rbd-nbd.cc

```diff
@@ -378,7 +377,6 @@ class NBDServer
 
   ~NBDServer()
   {
-    stop();
```

Comment:
I think the constructor can be removed totally in this case. Or `assert(!started)` added.